scripts/load_mentors.js

```diff
@@ -87,6 +88,9 @@ exports.handler = async (event) => {
   const mentorIds = new Set(mentors.map(({ id }) => id));
   const oldMentorIds = [];
 
+  const mentorMap = new Map(mentors.map((mentor) => [mentor.id, mentor]));
```

Comment:
Since we have a `Map` here, could we remove the obsolete `mentorIds` set and just use the map to check for the existence of the key?